src/mmw/js/src/modeling/models.js

```diff
@@ -40,6 +40,17 @@ var ProjectModel = Backbone.Model.extend({
         }
     },
 
+    set: function(attributes, options) {
+        if (attributes === 'active_scenario_slug') {
```

Comment:
`active_scenario_slug` is gone as of #249. There is also a custom method for setting the active scenario, `setActiveScenario`.